src/ui/handlers/message-ui-handler.ts

```diff
@@ -74,35 +68,27 @@ export abstract class MessageUiHandler extends AwaitableUiHandler {
   public showDialogue(
     text: string,
     _name?: string,
-    delay?: number | null,
-    callback?: Function | null,
-    callbackDelay?: number | null,
-    prompt?: boolean | null,
-    promptDelay?: number | null,
+    delay?: number,
+    callback?: VoidFunction,
+    callbackDelay?: number,
+    prompt?: boolean,
+    promptDelay?: number,
   ) {
-    this.showTextInternal(text, delay, callback, callbackDelay, prompt, promptDelay);
+    this.showTextInternal(text, { delay, callback, callbackDelay, prompt, promptDelay });
   }
 
   private showTextInternal(
     text: string,
-    delay?: number | null,
-    callback?: Function | null,
-    callbackDelay?: number | null,
-    prompt?: boolean | null,
-    promptDelay?: number | null,
+    { delay = 20, callback, callbackDelay, prompt, promptDelay }: ShowTextOptions = {},
```

Comment:
```suggestion
    { delay = 20, callback, callbackDelay = 0, prompt = false, promptDelay = 0 }: ShowTextOptions = {},
```
These defaults are already in place with the truthy/falsy checks in here; we might as well make them more explicit.